src/languages/en.ts

```diff
@@ -1511,7 +1511,9 @@ export default {
         checkHelpLine: 'Your routing number and account number can be found on a check for the account.',
         validateAccountError: {
             phrase1: 'Hold up! We need you to validate your account first. To do so, ',
-            phrase2: 'sign back in with a magic code',
+            phrase2: 'sign back in with a magic code ',
+            phrase3: 'or ',
+            phrase4: 'verify the account here',
```

Comment:
Let's change this to `verify your account here` (and also update the Spanish copy accordingly)